docs/index.rst

```diff
@@ -0,0 +1,18 @@
+Scality S3 Server
+==================
+
+.. _user-docs:
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Documentation
+
+   GETTING_STARTED
+   CONTRIBUTING
+   CLIENTS
+   DOCKER
```

Comment:
Since this section is already in the "Getting started" section, could we remove it from here?